Changes

```diff
@@ -1,5 +1,45 @@
 Release history for Zonemaster component Zonemaster-Backend
 
+v8.0.0 2021-12-03 (public release version)
+
+ [Breaking changes]
+ - Drops support for PostgreSQL version 9 and before (#892)
+ - Deprecates country code in RPC API calls (#796, #794)
```

Comment:
Deprecations aren't breaking changes. They're just an early announcements about upcoming breaking changes. The breaking change happens when the feature is actually removed. We should probably have a special section for deprecations.

Reply:
Updated.